pkg/sql/schemachanger/scplan/internal/opgen/opgen_column_name.go

```diff
@@ -26,15 +26,21 @@ func init() {
 				}),
 			),
 		),
+		toTransientAbsentLikePublic(),
 		toAbsent(
 			scpb.Status_PUBLIC,
 			to(scpb.Status_ABSENT,
 				emit(func(this *scpb.ColumnName) *scop.SetColumnName {
-					return &scop.SetColumnName{
+					op := &scop.SetColumnName{
 						TableID:  this.TableID,
 						ColumnID: this.ColumnID,
 						Name:     tabledesc.ColumnNamePlaceholder(this.ColumnID),
 					}
+					// If a name was provided for the transition to absent, override the placeholder.
```

Comment:
for my understanding: is this just setting the new column to the column column's name?